library/core/src/any.rs

```diff
@@ -657,7 +662,7 @@ impl hash::Hash for TypeId {
         // - It is correct to do so -- only hashing a subset of `self` is still
         //   with an `Eq` implementation that considers the entire value, as
         //   ours does.
-        (self.t as u64).hash(state);
```

Comment:
The comment above is now incorrect -- we are hashing the *upper* bits of the internal ID now. I don't think it matters much either way; can you update the comment to be accurate though?

Reply:
Fixed by just following the comment.